src/peer_review/tests/test_peer_review_invite_view.py

```diff
@@ -0,0 +1,267 @@
+from rest_framework.test import APITestCase
+from user.tests.helpers import (
+    create_random_default_user,
+    create_moderator,
+)
+from hub.tests.helpers import create_hub
+from peer_review.tests.helpers import create_peer_review_request
+from peer_review.models import PeerReviewRequest
+from user.models import Organization, User
+
+
+class PeerReviewInviteViewTests(APITestCase):
+    def setUp(self):
```

Comment:
```suggestion
    def set_up(self):
```

Reply:
wasn't sure if there was some pattern for camel casing this since i see some camel casing in test files?

Reply:
`APITestCase` has an abstract `setUp` method.